src/api/index.ts

```diff
@@ -349,37 +366,60 @@ export const withdraw = async (code: TokenCode, amount: Balance, account?: Accou
 */
 export const getSellerOngoingAuctions = async (tokensJSON: DefaultTokenList, account: Account): Promise<any[]> => {
```

Comment:
@Velenir this function here is my biggest concern - it's messy. Feel free to demolish it/optimise it if you're feeling up to it. Works though /shrug